pkg/tls/tlsmanager.go

```diff
@@ -408,6 +408,9 @@ func buildTLSConfig(tlsOption Options) (*tls.Config, error) {
 		}
 	}
 
+	// Set SessionTicketsDisabled if set in the config
+	conf.SessionTicketsDisabled = tlsOption.DisableSessionTickets
```

Comment:
Could you move this initialisation to line https://github.com/traefik/traefik/pull/11609/files#diff-3f7f33621af8b7e588a5375221f22cf500b5d598ceb3f5fbb8be623c94defd41R328?